src/components/cart/cart.css

```diff
@@ -0,0 +1,166 @@
+.cart-container {
+  gap: 2rem;
+  padding: 1.5rem;
+}
+
+.cart-items {
+  border-right: 1px solid var(--gray-color);
+  padding: 1rem;
+  row-gap: 2rem;
+}
+
+.cart-horizontal-wide {
+  border-radius: 5px;
+  border: 1px solid var(--gray-color);
+  display: flex;
+  flex-direction: column;
+  height: 12.3rem;
+  margin: 0rem 1rem;
+  width: 22rem;
+}
+
+.cart-img-container {
+  height: 9.5rem;
+  margin: 5px;
+  width: 10rem;
+}
+
+.cart-img-and-content {
+  display: flex;
+}
+
+.cart-img-container > img {
+  height: 100%;
+  object-fit: cover;
+  width: 100%;
+}
+
+.cart-content {
+  display: flex;
+  flex-direction: column;
+  margin: 5px;
+}
+
+.cart-title {
+  padding: unset;
+}
+
+.cart-buttons {
+  border-top: 1px solid var(--gray-color);
+  display: flex;
+}
+
+.cart-button {
+  align-items: center;
+  border: unset;
+  display: inline-flex;
+  height: 1rem;
+  justify-content: center;
+  margin: 0.5rem 0rem;
+  padding: 6px;
+  width: 100%;
+  font-weight: bold;
+}
+
+.cart-buttons:first-child {
+  width: 80%;
+}
+
+.cart-quantity-buttons {
+  margin: 0.5rem;
+}
+
+.button-increase,
+.button-decrease {
+  align-items: center;
+  cursor: pointer;
+  display: inline-flex;
+  height: 2rem;
+  justify-content: center;
+  padding: 5px;
+  width: 2rem;
+}
+
+.quantity-display {
+  color: var(--default-color);
+  font-weight: bold;
+  margin: 1rem;
+}
+
+.cart-items-and-price {
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.cart-price {
+  align-self: flex-start;
+  display: flex;
+  flex-direction: column;
+  font-size: 0.8rem;
+  padding-top: 1rem;
+  margin-left: 1rem;
+  row-gap: 2px;
+}
+
+.price-row {
+  display: flex;
+  row-gap: 1rem;
+}
+
+.price-row > div {
+  display: inline;
+  width: 20%;
+  padding: 5px 0px;
+}
+
+.price-row > .display-left {
+  width: 20ch;
+}
+
+.font-bold {
+  font-weight: bold;
+}
+.display-right {
+  align-self: flex-end;
+  flex-grow: 1;
+  padding-left: max(2rem, 2rem);
+}
+
+.empty-cart-icon {
+  color: var(--disabled-color);
+  padding: 2rem;
+}
+
+.cart-empty-container {
+  display: flex;
+}
+
+.cart-empty-container > .btn {
+  height: 2.5rem;
+  padding: 0.5rem;
+  width: fit-content;
+}
+
+.order-button,
+.coupon-button {
+  width: 100%;
+  height: 2rem;
+  margin: unset;
+  margin: 0.5rem 0;
+}
+
+.coupon-button:hover {
+  background-color: var(--gray-color);
+  color: unset;
+  /* width: 100%; */
```

Comment:
Small suggestion: Can remove this commented code later on as refactoring

Reply:
removed the comment, thankyou for pointing it out